src/main/java/teammates/common/util/Const.java

```diff
@@ -1379,6 +1386,8 @@ private Const() {
         public static final String FEEDBACK_SESSION_REMINDERSSENT =
                 "Reminder e-mails have been sent out to those students and instructors. "
                 + "Please allow up to 1 hour for all the notification emails to be sent out.";
+        public static final String FEEDBACK_SESSION_ACCESS_LINKS_RESENT =
+                "Recent feedback session access links have been sent to the student who requested for a resend.";
```

Comment:
Actually can remove the `to the student who requested for a resend` part; it'll be weird for the student to receive such a message after submitting.